pressbooks/pressbooks
# Add Sentry integration for Backend and Frontend

Related issue: https://github.com/pressbooks/private/issues/374

This change implements a new Sentry Integration to PB, and also add a JavaScript observer to monitor performance and catch front end issues.
Details about how it works and documentation: https://docs.google.com/document/d/1hg35gRWlxcQqC2DElrBi2btLFQQXvMgs-NhgvmnMqpA/edit

### How to test
- Create and set the Environment variables following [docs](https://docs.google.com/document/d/1hg35gRWlxcQqC2DElrBi2btLFQQXvMgs-NhgvmnMqpA/edit) in your local environment (make sure `ENABLE_SENTRY=1`
- Force any error or exception in any enable plugin or theme within your code
- Check [Sentry pressbooks project](https://sentry.io/organizations/pressbooks/issues) issues and make sure the issue is there.

_Note there are new ENV variables for sentry and also we can  delete the previous one in Staging after apply this change._
